services/api-server/tests/unit/api_functions/conftest.py

```diff
@@ -0,0 +1,358 @@
+    )
+    mock_wb_api_server_rpc.patch(
+        "servicelib.rabbitmq.rpc_interfaces.webserver.functions.functions_rpc_interface.get_function",
+        backend_function_register.get_function,
+    )
+    mock_wb_api_server_rpc.patch(
+        "servicelib.rabbitmq.rpc_interfaces.webserver.functions.functions_rpc_interface.run_function",
+        backend_function_register.run_function,
+    )
+    mock_wb_api_server_rpc.patch(
+        "servicelib.rabbitmq.rpc_interfaces.webserver.functions.functions_rpc_interface.list_functions",
+        backend_function_register.list_functions,
+    )
+    mock_wb_api_server_rpc.patch(
+        "servicelib.rabbitmq.rpc_interfaces.webserver.functions.functions_rpc_interface.delete_function",
+        backend_function_register.delete_function,
+    )
+    mock_wb_api_server_rpc.patch(
+        "servicelib.rabbitmq.rpc_interfaces.webserver.functions.functions_rpc_interface.register_function_job",
+        backend_function_register.register_function_job,
+    )
+    mock_wb_api_server_rpc.patch(
+        "servicelib.rabbitmq.rpc_interfaces.webserver.functions.functions_rpc_interface.get_function_job",
+        backend_function_register.get_function_job,
+    )
+    mock_wb_api_server_rpc.patch(
+        "servicelib.rabbitmq.rpc_interfaces.webserver.functions.functions_rpc_interface.list_function_jobs",
+        backend_function_register.list_function_jobs,
+    )
+    mock_wb_api_server_rpc.patch(
+        "servicelib.rabbitmq.rpc_interfaces.webserver.functions.functions_rpc_interface.delete_function_job",
+        backend_function_register.delete_function_job,
+    )
+    mock_wb_api_server_rpc.patch(
+        "servicelib.rabbitmq.rpc_interfaces.webserver.functions.functions_rpc_interface.register_function_job_collection",
+        backend_function_register.register_function_job_collection,
+    )
+    mock_wb_api_server_rpc.patch(
+        "servicelib.rabbitmq.rpc_interfaces.webserver.functions.functions_rpc_interface.get_function_job_collection",
+        backend_function_register.get_function_job_collection,
+    )
+    mock_wb_api_server_rpc.patch(
+        "servicelib.rabbitmq.rpc_interfaces.webserver.functions.functions_rpc_interface.list_function_job_collections",
+        backend_function_register.list_function_job_collections,
+    )
+    mock_wb_api_server_rpc.patch(
+        "servicelib.rabbitmq.rpc_interfaces.webserver.functions.functions_rpc_interface.delete_function_job_collection",
+        backend_function_register.delete_function_job_collection,
+    )
+    mock_wb_api_server_rpc.patch(
+        "servicelib.rabbitmq.rpc_interfaces.webserver.functions.functions_rpc_interface.update_function_title",
+        backend_function_register.update_function_title,
+    )
+    mock_wb_api_server_rpc.patch(
+        "servicelib.rabbitmq.rpc_interfaces.webserver.functions.functions_rpc_interface.update_function_description",
+        backend_function_register.update_function_description,
+    )
```

Comment:
I see the point of what you are doing but let me suggest soemthing different that removes the entire client you added above


```suggestion
@pytest.fixture()
def mock_handler_in_functions_rpc_interface(
    mock_wb_api_server_rpc: MockerFixture,
) -> Callable[[str, Any], None]:
    def _(handler_name: str, return_value: Any) -> None:
        mock_wb_api_server_rpc.patch(
            f"servicelib.rabbitmq.rpc_interfaces.webserver.functions.functions_rpc_interface.{handler_name}",
            return_value=return_value,
        )

    return _
```

Reply:
I got rid of the entire function register and am using your fixture